internal/chain/beacon/cache.go

```diff
@@ -3,9 +3,9 @@ package beacon
 import (
 	"bytes"
 	"encoding/binary"
-
 	"github.com/drand/drand/crypto"
-	"github.com/drand/drand/log"
+
```

Comment:
lint?

Reply:
hmm even after running GCI this doesn't seem to change